releasenotes/notes/istioctl-mesh-install.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+releaseNotes:
+- |
+  **Deprecated** `istioctl manifest install` command is deprecated and will be removed in Istio 1.9. Use `istioctl install` instead.
```

Comment:
`manifest apply` was deprecated in `1.6` and removed with `1.7`. see [here](https://github.com/istio/istio.io/pull/7502/files#diff-eb6e9d0c5d20f9923cc19b8b24fc8f14R43).  So no need for this release note.